lang/llvm-10/Portfile

```diff
@@ -287,7 +287,9 @@ if {[lsearch -exact $PortInfo(depends_build) port:cctools] != -1} {
 }
 
 # use the system python27 if present
-if {${os.platform} eq "darwin" && ${os.major} >= 11} {
+if {${os.platform} eq "darwin" && ${os.major} >= 22} {
+    set pythonfullpath   /usr/bin/python3
+} elseif {${os.platform} eq "darwin" && ${os.major} >= 11} {
```

Comment:
Would it be better to depend on MacPorts python3 and not worry about this?

Reply:
Old OSes can happily run Python 3 so there's no need for them to have a build dependency on Python 2.